src/pages/gen1/index.tsx

```diff
@@ -8,7 +8,7 @@ import LinkCards from '@/components/LinkCards';
 import PlatformFeatureList from '@/components/FeatureLists/PlatformFeatureList';
 
 export const meta = {
-  title: 'Overview',
+  title: 'Amplify Docs (Gen 1)',
```

Comment:
In prod, we do something like this for Gen 2, and so I updated this title to have it reflect in the breadcrumbs